themes-default/slim/views/vue-components/select-list-ui.mako

```diff
@@ -174,6 +168,12 @@ Vue.component('select-list', {
         },
         csv() {
             this.syncValues();
+        },
+        listItems: {
+            handler: function() {
+                this.constructList();
+            },
+            deep: true
```

Comment:
I don't think you even need `deep: true`. (it works without)
You're sending it a list of strings, or a string, so in any case no objects are involved.